canonical/lxd-ui
# Allow login with oidc WD-4314

## Done

- add oidc login support
- overhaul login page (two versions with and without oidc support by the backend)
- added logout when oidc login
- added oidc keys to the settings

Fixes WD-4314

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check oidc login, see [tutorial](https://github.com/canonical/lxd-ui/wiki/Setup-oidc-login) for the setup
